docs/variants/talos_2/initial-deployment.md

```diff
@@ -1,4 +1,4 @@
-# Insurgo Dasharo on Talos II - installation manual
+# Insurgo Dasharo on Talos II - initial deployment
```

Comment:
Take a look on the ASUS or Protectli or Optiplex documentation - we should keep naming headings.

Reply:
I'd take a look at Protectli VP4, as it was the latest release: https://docs.dasharo.com/variants/protectli_vp46xx/initial-deployment/

There is no point in repeating platform name in each document title. Just `Initial deplyment` should suffice.

Reply:
Sure, I renamed it